.gitignore

```diff
@@ -9,6 +9,7 @@
 /bower_components
 
 # misc
+/.idea
```

Comment:
BTW, I've found that it's often more convenient/clean to put IDE-related git ignore entries in a global (per machine) file: https://stackoverflow.com/questions/7335420/global-git-ignore#7335487